sips/minutes/_posts/2016-08-20-sip-20th-september-minutes.md

```diff
@@ -0,0 +1,238 @@
+
+The following proposal was accepted:
+
+* SIP-27: Trailing commas
+
+The following proposal was rejected:
+
+* SIP-26: Unsigned Integer Data Types
+
+The following proposal will have a follow-up evaluation:
+
+* SIP-21: Spores
+
+The following proposals were numbered:
+
+* SIP-28: Inline (part of Scala Meta SIP)
+* SIP-29: Meta (part of Scala Meta SIP)
+
+## Date, Time and Location
+
+The meeting took place at 5:00pm Central European Time / 8:00am Pacific Daylight
+Time on Tuesday, September 20th, 2016 via Google Hangouts.
+
+Minutes were taken by Jorge Vicente Cantero, acting secretary.
+
+## Attendees
+
+Attendees Present:
+
+* Martin Odersky ([@odersky](github.com/odersky)), EPFL
+* Seth Tisue ([@SethTisue](github.com/SethTisue)), EPFL
+* Iulian Dragos ([@dragos](github.com/dragos)), Independent
+* Heather Miller ([@heathermiller](github.com/heathermiller)), Scala Center
+* Sébastien Doeraene ([@sjrd](github.com/sjrd)), EPFL
+* Eugene Burmako ([@xeno-by](github.com/xeno-by)), EPFL
+* Josh Suereth ([@jsuereth](github.com/jsuereth)), Google
+* Jorge Vicente Cantero ([@jvican](github.com/jvican)), Process Lead
+
+## Guests
+
+* Dale Wijnand ([@dwijnand](github.com/dwijnand)), author of SIP-27
+
+## Apologies
+
+* Andrew Marki ([@som-snytt](github.com/som-snytt)), independent
+* Adriaan Moors ([@adriaanm](github.com/adriaanm)), Lightbend
+
+## Proceedings
+### Opening Remarks
+
+As acting Process Lead, Jorge gave the welcome to Iulian Dragos, who has just
+joined the Committee. He's a natural fit for the Committee, because of
+his historical significance in the evolution of Scala and his vast experience in
+the current Scala compiler (scalac).
+
+Following to his introduction, he described the main points to the agenda and
+proceed to introduce the discussions.
+
+### Discussion of SIP-NN: Scala Meta SIP
+
+Josh and Iulian are the reviewers of the proposal. Josh starts explaining what
+the proposal is about. Scala Meta is split into two main parts: inline
+and meta. The purpose of this meeting is to number the proposal and kick start a
+previous discussion. Jorge proposes to vote, and Iulian points out that before
+voting he considers that inline and meta should be separate.
+
+Sébastien argues that it may not be possible, since they are tightly
+interconnected and meta relies heavily on a concrete behaviour of inline. Martin
+tends to agree with Iulian, and Josh claims that, if possible, studying both
+parts separately would be ideal. Sébastien is not sure that they can be
+reorganized to be completely orthogonal. Eugene confirms that the dependency
+between inline and meta is very big. For him, studying them separately is a
+concern because there could be a point where meta cannot happens because of the
+way inline was designed.
+
+Seth asks for further information on the interdependency. Sébastien argues that,
+conventionally, metaprogramming has relied on the eager execution of the formal
```

Comment:
If I said such a thing, it was downright wrong. Existing scala.reflect macro only ever see the *non* eagerly-executed actual arguments, since they see their Trees instead.